packages/flex-plugin-scripts/src/config/index.ts

```diff
@@ -56,7 +57,11 @@ const getConfiguration = async <T extends ConfigurationType>(
       try {
         return require(getPaths().app.webpackConfigPath)(config, args);
       } catch (exception) {
-        await emitDevServerCrashed(exception);
+        if (script === 'start') {
```

Comment:
Mmm, better use `isServerRunning` (or `isClientConnected`, i.e. if one is running) from the `flex-plugins-webpack` ipc-server module instead . Don't pass `script` here.